modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -120,11 +120,11 @@ func (suite *KeeperTestSuite) TestQueryIncentivizedPackets() {
 				fee3 := types.NewIdentifiedPacketFee(channeltypes.NewPacketId(ibctesting.FirstChannelID, transfertypes.PortID, 3), fee, refundAcc.String(), []string(nil))
 
 				expPackets = []*types.IdentifiedPacketFee{}
-				expPackets = append(expPackets, fee1, fee2, fee3)
+				expPackets = append(expPackets, &fee1, &fee2, &fee3)
 
 				suite.chainA.GetSimApp().IBCFeeKeeper.SetFeeEnabled(suite.chainA.GetContext(), transfertypes.PortID, ibctesting.FirstChannelID)
 				for _, p := range expPackets {
-					suite.chainA.GetSimApp().IBCFeeKeeper.EscrowPacketFee(suite.chainA.GetContext(), p)
+					suite.chainA.GetSimApp().IBCFeeKeeper.EscrowPacketFee(suite.chainA.GetContext(), *p)
```

Comment:
@colin-axner @damiannolan  should I just update the queries to no longer return a pointer as well?

Reply:
The grpc query server endpoints will always use a pointer req/resp type. I'm not sure there's anyway to not do that, unless there's some gogoproto magic. But all of the grpc queries in ibc-go/sdk return pointer response types afaik so best to keep it consistent.